app.js

```diff
@@ -1,18 +1,19 @@
 const express = require("express");
+const Job = require("./models/Job");
 require("express-async-errors");
-
 const app = express();
 
 app.set("view engine", "ejs");
 app.use(require("body-parser").urlencoded({ extended: true }));
 
-require("dotenv").config(); // to load the .env file into the process.env object
-const session = require("express-session");
+require("dotenv").config();
+const session = require("express-session");   
 const MongoDBStore = require("connect-mongodb-session")(session);
-const url = process.env.MONGO_URI;
+const csurf = require("csurf"); // CSRF protection
```

Comment:
This anti-csrf package is deprecated:
[https://www.npmjs.com/package/csurf](https://www.npmjs.com/package/csurf)

Ideally, you'd use the host-csrf package that was recommended in the lesson:
[https://www.npmjs.com/package/host-csrf](https://www.npmjs.com/package/host-csrf)
which functions nearly identically to the csurf package.